README.md

```diff
@@ -1,43 +1,153 @@
-# Project 
-Exercise tracker
+<!-- PROJECT -->
+  <p align="center">
+   Phisical Fitness
```

Comment:
Should this show up as the largest header (basically an H1?) Also, note that 'Physical Fitness' is the proper english spelling.